### ISSUE TITLE
Accessing the Adanet loss for each candidate subnetwork?

### ISSUE THREAD

**Issue by nicholasbreckwoldt** [tensorflow/adanet#120]
Hi there,

I would like to be able to access and record the individual Adanet losses returned by the _adanet.Evaluator_ during evaluation of each subnetwork (i.e. candidate) within a given iteration. 

Is this currently possible, and if so, how would one go about accessing these losses?

**Comment by EugenHotaj** [comment]
Hi @nicholasbreckwoldt, 

As far as I know, there is no out of the box way to get the subnetwork losses from the AdaNet estimator. For example, when calling estimator.evaluate(), only the metrics for the final ensemble are returned.

However, if you're only interested in tracking the losses during training, we do write them out as summaries for each subnetwork, which can be viewed via Tensorboard. If, instead, you're interested in computing something based on these losses, this would require modifying the AdaNet code to use the losses however you need to.
